src/index.css

```diff
@@ -1,70 +1,70 @@
-:root {
-  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: 400;
+/*:root {*/
```

Comment:
I'd keep :root styling and remove the rest
It's always better to remove code instead of commenting it out

Reply:
done